Makefile

```diff
@@ -191,16 +189,16 @@ coverage-report: diff_grep:=$(shell echo $(diff_sed) | tr ' ' '\n' | grep \.py |
 coverage-report:
 	@docker-compose run --rm web coverage report -i --skip-covered $(diff_grep) | grep -v "NoSource:"
 
-coverage-html:
+coverage-html-report:
 	@docker-compose run --rm web coverage html --omit="*/tests/*"
 
-coverage-html-open: coverage-html
+coverage-html: coverage
```

Comment:
Not sure this is a correct dependency. I might want just to open it, without running it again

Reply:
Sounds like you are proposing a different target that just opens the latest coverage report.  I still think we want one that runs the coverage and then opens the result (if you like that kind of thing).  We could do that I suppose.  Personally when I want to do that I just go to File => Open in the browser.  The only reason I implemented "coverage-html" is that Shankar did it on impact.  Personally I'm fine with just opening it from the browser.

Reply:
I guess I am too. Doesn't matter much.